src/lightning/data/CHANGELOG.md

```diff
@@ -5,6 +5,34 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
+## [2.1.4] - 2024-01-31
```

Comment:
Can you remove these changes, @tchaton asked me not to include the list of changes.

Reply:
Let's ask @lantiga